server/viame_server/viame_detection.py

```diff
@@ -166,6 +166,16 @@ def get_detection(self, folder):
         file = Item().childFiles(detectionItems[0])[0]
         if "csv" in file["exts"]:
             return viame.load_csv_as_tracks(file)
+        else:
+            # There might be up to 3 yamls
+            allFiles = [Item().childFiles(item)[0] for item in detectionItems[:3]]
```

Comment:
I'm am down to do that...not sure if I know how to. Would that occur when they're being uploaded?

Reply:
Yes, this would require a change in how they're uploaded, and would possibly be a large change to the client upload code. If that is not something we want to address right now, I believe something needs to be done regarding how the items are handled, if all three `yaml` files are supposed to represent one combined detection.

Reply:
Discussed with @mattrix27, we decided to address this in a follow up PR, which I will create an issue for.

Reply:
https://github.com/VIAME/VIAME-Web/issues/242